modules/openapi-generator/src/main/resources/python-nextgen/model_generic.mustache

```diff
@@ -241,7 +241,42 @@ class {{classname}}({{#parent}}{{{.}}}{{/parent}}{{^parent}}BaseModel{{/parent}}
             {{#isMap}}
             {{^items.isPrimitiveType}}
             {{^items.isEnumOrRef}}
-            "{{{name}}}": dict((_k, {{{items.dataType}}}.from_dict(_v)) for _k, _v in obj.get("{{{baseName}}}").items()) if obj.get("{{{baseName}}}") is not None else None{{^-last}},{{/-last}}
+            {{#items.isContainer}}
+            {{#items.isMap}}
+            "{{{name}}}": dict(
+                (_k, dict(
+                    (_ik, {{{items.items.dataType}}}.from_dict(_iv))
+                        for _ik, _iv in _v.items()
+                    )
+                    if _v is not None
+                    else None
+                )
+                for _k, _v in obj.get("{{{baseName}}}").items()
+            )
+            if obj.get("{{{baseName}}}") is not None
+            else None{{^-last}},{{/-last}}
+            {{/items.isMap}}
+            {{#items.isArray}}
+            "{{{name}}}": dict(
+                (_k, [(_ik, {{{items.items.dataType}}}.from_dict(_iv))]
+                    for _ik, _iv in _v.items()
+                    if _v is not None
+                    else None
+                )
```

Comment:
This change fails to parse for me.

Instead what works, is:
```python
                (_k, [(_ik, {{{items.items.dataType}}}.from_dict(_iv))
                    if _v is not None
                    else None
                    for _ik, _iv in _v.items()
                ])
```

Reply:
I also can't find any test case for that scenario.

Reply:
can you please file a PR with the suggested fix? we'll try to add a test or two on top of your fix.

Reply:
fixed in https://github.com/OpenAPITools/openapi-generator/pull/15844